app/mixins/marketing-route-mixin.js

```diff
@@ -0,0 +1,19 @@
+import { get } from '@ember/object';
+import Mixin from '@ember/object/mixin';
+import { inject as service } from '@ember/service';
+
+export default Mixin.create({
+  siteFooter: service(),
+
+  actions: {
+    didTransition() {
+      get(this, 'siteFooter').enlarge();
+      return true;
+    },
+
+    willTransition() {
+      get(this, 'siteFooter').reduce();
```

Comment:
Just on the basis of `Array.prototype.reduce()` being a thing, how would you feel about renaming this to `.shrink()`?

Reply:
I would agree. Can you make this change and merge?